testApi/src/main/scala/africa/absa/testing/scapi/rest/response/action/ExtractJsonResponseAction.scala

```diff
@@ -77,17 +81,17 @@ object ExtractJsonResponseAction extends ResponsePerformer {
    * @param listIndex            The index in the JSON array from which to extract the string.
    * @param jsonKey              The key in the JSON object from which to extract the string.
    * @param runtimeCacheLevel    The expiration level to use when storing the extracted string in the runtime cache.
-   * @return Boolean indicating whether the string extraction and caching operation was successful.
+   * @return A Try[Unit] indicating whether the string extraction and caching operation was successful or not.
    */
-  def stringFromList(response: Response, cacheKey: String, listIndex: Int, jsonKey: String, runtimeCacheLevel: String): Boolean = {
+  private def stringFromList(response: Response, cacheKey: String, listIndex: Int, jsonKey: String, runtimeCacheLevel: String): Try[Unit] = {
     try {
```

Comment:
why `try` to return `Success()`, why not just `Try {code-that-may-throw-otherwise we end up with Success)`?

Reply:
Fixed in commit a15a4201defe36c53ed748e66e4bad34aa6372ac.